drivers/overlay/overlay.go

```diff
@@ -266,18 +283,27 @@ func Init(home string, options graphdriver.Options) (graphdriver.Driver, error)
 	}
 
 	if opts.mountProgram != "" {
+		if unshare.IsRootless() && isNetworkFileSystem(fsMagic) && opts.forceMask == nil {
+			m := os.FileMode(0700)
+			opts.forceMask = &m
+			logrus.Debugf("Network file system detected as backing store.  Enforcing overlay option `force_mask=\"%o\"`.", m)
```

Comment:
This should be at the Info level, or should it be Warn?  Would it be better for users to configure this themselves within the storage.conf, and warn them that we did it automatically.

Reply:
I had it at Warn level first, but seemed annoying.  I can switch it back to Warnf

Reply:
reverted to Warning